todoman/ui.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, todo, lists, formatter):
         else:
             dtstart = ''
 
+        if todo.priority:
```

Comment:
The `else` branch is missing here. `todo edit` crashes on tasks without priority. This should work:

    ...
    else:
        priority = ''